jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -72,7 +83,7 @@ pipeline {
                 script {
                     def changes = sh(script: 'git log ${productBranch}..origin/${baseBranch} --pretty=o | wc -l', returnStdout: true)
                     if ((changes as int) > 0) {
-                        sh 'git push origin ${productBranch} --force'
+                        // sh 'git push origin ${productBranch} --force'
```

Comment:
This needs to be uncommented before merging.

Reply:
Since you aren't committing anything, this would do nothing anyway

Reply:
Yeah. Just being over cautious.